### PR TITLE
Form

### PR DESCRIPTION
**Pull request by matschi3**
finished for now // pretty raw

### CODE REVIEW

File: bookmark.html [hunk 1]
```diff
@@ -16,7 +16,7 @@ <h1>QUIZZER</h1>
     <!-- FOOTER -->
     <footer>
       <nav class="navmenu">
-        <a href="./index.html" class="navitem">
+        <a href="./index.html" class="navitem" aria-label="Go to start site">
```
Comment (julialoeschel) [inline bookmark.html:19]: great you used aria-label here!!!

File: form.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    <title>QUIZZER - Cool Quiz</title>
+    <link rel="stylesheet" href="./styles.css" />
+    <script src="./index.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <h1>QUIZZER</h1>
+    </header>
+    <main>
+      <form aria-labelledby="form-legend">
```
Comment (julialoeschel) [inline form.html:16]: if you want it to be labeled with `Add a new unique Question here:` make sure to refere to the exact id of your legend. 
`formLegend != form-legend`

File: index.html [hunk 3]
```diff
@@ -14,6 +14,7 @@ <h1>QUIZZER</h1>
     </header>
     <main>
       <section class="qcard_container">
```
Comment (julialoeschel) [inline index.html:16]: I know its obvious, but as more your code grows - the less obvious it is: try to wirte the words out. always. your future self will thank you.

**Review by julialoeschel — COMMENTED** [review]
:octocat: 
great job there!!!
I just left some reviews for you, you can consider them in your future porjects.

**Comment by matschi3** [comment]
fixed all in fix-branch.
Thanks to @julialoeschel

**Pull request merged**